packages/api/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
looks like you might have run npm start from `api` 

lets delete the `api/public` folder and everything in it as all that lives on the app/ side of things

Reply:
I ran npm start from api response is good no errors i did delete migrations and public from api. Build still fails in PR.